evergreen/fetch-csfle-library.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Find the version of python on the system.
+#
+# Environment variables used as input:
+#   OS                                               The current operating system
+#
+# Environment variables produced as output:
+#   MONGODB_CSFLE_PATH                               The MONGODB_CSFLE_PATH path
+
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with an error if any of the commands fail
+
+
+PYTHON=$(OS=${OS} ${PROJECT_DIRECTORY}/evergreen/get-python-path.sh) # > /dev/null
```

Comment:
I guess this is only for debug purpose: `# > /dev/null`. If so, then let's remove it